mwdb/model/group.py

```diff
@@ -37,6 +46,10 @@ def pending_group(self):
 
     @property
     def immutable(self):
+        """
```

Comment:
A reflection after reading this PR - it looks like "immutable" is now an important property of a group. Maybe we should consider exposing it a bit more in the UI? For example, by adding small "lock" next to the group name in group admin view?

WDYT? (of course not in this PR)